conformance/packages/conformance-tests/src/resolver/dnssec/scenarios/insecure.rs

```diff
@@ -0,0 +1,84 @@
+use std::net::Ipv4Addr;
+
+use dns_test::client::{Client, DigSettings};
+use dns_test::name_server::NameServer;
+use dns_test::record::{Record, RecordType};
+use dns_test::zone_file::SignSettings;
+use dns_test::{Network, Resolver, Result, TrustAnchor, FQDN};
+
+// in this DNS network all zones except one are signed. and importantly, the referral to the
+// unsigned zone (the NS+A records in the parent zone) is also signed
+//
+// a validating resolver should not respond with SERVFAIL to queries about the unsigned zone because
+// the security status of the whole zone is "Insecure", not "Bogus"
+#[test]
+#[ignore]
+fn unsigned_zone() -> Result<()> {
+    let expected_ipv4_addr = Ipv4Addr::new(1, 2, 3, 4);
+    let needle_fqdn = FQDN("example.unsigned.com.")?;
```

Comment:
This should use the testing TLD from #2359, right?

Reply:
once that PR is merged, this domain should be updated, yes.